src/types/value/repeatable.ts

```diff
@@ -0,0 +1,9 @@
+import type { AnyRepeatableField, FieldState } from "./types"
+
+/**
+ * A list of repeatable fields.
+ */
+export type Repeatable<
+	Fields extends Array<AnyRepeatableField> = Array<AnyRepeatableField>,
```

Comment:
I don't think this is right, because when `AnyRepeatableField` is composed of more than one field the array won't be homogeneous.